lib/sproxyd.js

```diff
@@ -147,9 +147,8 @@ class SproxydClient {
 
     _failover(method, stream, size, key, tries, log, callback, params) {
         const args = params === undefined ? {} : params;
-        const value = key || '[data]';
         let counter = tries;
-        log.info('request', { method, value, args, counter });
+        log.info('request', { method, key, args, counter });
```

Comment:
While we are here we should update the log message to be more specific. Something like
`sending request to sproxyd`